### DISCUSSION TITLE
Connection to Simulator failed.

### DISCUSSION THREAD

**Discussion by wdepu** [mpaperno/MSFSTouchPortalPlugin#46] [Q&A] [answered]
I can not connect in any way to the simulator but with WASimClient no problem. 
I need to edit some config files?

I hope the logs can help.

Thanks
Marco
[MSFSTouchPortalPlugin20221205.log](https://github.com/mpaperno/MSFSTouchPortalPlugin/files/10156272/MSFSTouchPortalPlugin20221205.log)
[WASimClient.log](https://github.com/mpaperno/MSFSTouchPortalPlugin/files/10156276/WASimClient.log)

**Comment by mpaperno** [comment, marked as answer]
Hi Marco,

Are you sing a custom SimConnect.cfg file?  Based on this log line...

> Copied SimConnect.cfg file from user settings folder. 

The "user settings folder" its referring to would be in your `%APPDATA%\MSFSTouchPortalPlugin\SimConnect.cfg`

Do you have anything configured in the plugin's Settings (in Touch Portal -> Settings => Plug-ins) for "SimConnect.cfg index" setting?

Do you have a custom SimConnect.xml set up?  ([Item 3 on this page](https://github.com/mpaperno/MSFSTouchPortalPlugin/wiki/Multiple-Touch-Portal-Device-Setup#how-to-use-multiple-touch-portal-devices) has details on where to find that based on your game version.)

I would expect that the WASimClient would be using the same network configuration as your main SimConnect setup (at least that is the intention) but perhaps something is amiss there.  I see from the log that the Client is using the default config ID of "-1" which literally means "the default SimConfig from either cfg file or system default).

Hope some of that makes sense... and helps  :)    LMK what you find.

Best,
-Max

**Reply by wdepu** [reply]
Hi Max,
many many thanks.
Issue solved, I added the new port in SimConnect.xml as in the example.

Thanks again you were very fast and kind.

Marco
Italy

**Reply by mpaperno** [reply]
Perfect, glad to help!  Thanks for letting me know, and also thanks for those logs... you saved us both a lot of time  👍🏼 

Ciao!
-Massimo
(spent some time in Italy... "x" just doesn't work :)
